components/eventsrv/src/config.rs

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2017 Chef Software Inc. and/or applicable contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Configuration for a Habitat OriginSrv service
+
+use core::config::ConfigFile;
+
+use error::Error;
+
+#[derive(Debug, Deserialize)]
+#[serde(default)]
+pub struct Config {
+    pub consumer_port: u16,
+    pub producer_port: u16,
+}
+
+impl Default for Config {
+    fn default() -> Self {
+        Config {
+            consumer_port: 9689,
```

Comment:
I just chose a couple of "unassigned ports" (no lists of well known ports contain a record for these) for these. Feel free to change them if you want! You'll just need to make sure you update any of your test client stuff to respect this